testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -198,6 +203,39 @@ class MiniAppDisplayActivity : BaseActivity() {
             )
     }
 
+    private fun setupMiniAppCameraPermissionDispatcher(){
+        val miniAppCameraPermissionDispatcher = object : MiniAppCameraPermissionDispatcher {
+            override fun getCameraPermission(
+                onSuccess: (isGranted: Boolean) -> Unit,
+                onError: (message: String) -> Unit
+            ) {
+                if (ContextCompat.checkSelfPermission(
+                        this@MiniAppDisplayActivity,
+                        Manifest.permission.CAMERA
+                    ) == PackageManager.PERMISSION_GRANTED
+                ) {
+                    onSuccess(true)
+                } else {
+                    onError("Camera Permission not granted")
+                }
+            }
+
+            override fun requestCameraPermission(
+                miniAppPermissionType: MiniAppDevicePermissionType,
+                callback: (isGranted: Boolean) -> Unit
```

Comment:
We can rename `callback` to `permissionRequestCallback` to match with corresponding parameter in the SDK interface.